docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,355 @@
+# Running BigDL-Orca Program on YARN
+
+This tutorial provides a step-by-step guide on how to run BigDL-Orca programs on Apache Hadoop (YARN) clusters, using a [PyTorch Fashin-MNIST program](https://github.com/intel-analytics/BigDL/tree/main/docs/docs/tutorials/tutorial_example/Fashion_MNIST/) as a working example.
+
+# 1. Key Concepts
+## 1.1 Init_orca_context
+A BigDL Orca program usually starts with the initialization of OrcaContext. For every BigDL Orca program, you should call `init_orca_context` at the beginning of the program as below:
+
+```python
+from bigdl.orca import init_orca_context
+
+init_orca_context(...)
+```
+
+In `init_orca_context`, you could specify necessary runtime configurations for the Orca program. By specifying `cluster_mode`, `init_orca_context` will create or get an OrcaContext with optimized configurations for BigDL performance, which will automatically provision the runtime Python environment and distributed execution engine on the yarn cluster.
+
+For more details, please see [OrcaContext](https://bigdl.readthedocs.io/en/latest/doc/Orca/Overview/orca-context.html) .
+
+## 1.2 Yarn-Client & Yarn-Cluster
+The difference between yarn-client and yarn-cluster is where you run your Spark driver. 
+
+For yarn-client, the Spark driver will run on the node where you start Python, while for yarn-cluster the Spark driver will run on a random node in the YARN cluster. So if you are running with yarn-cluster, you should change the data loading from local file to a network file system (e.g. HDFS).
```

Comment:
"For yarn-client, the Spark driver will run on the node where you start Python"

Reply:
I don't it is recommended to use local files in production even when running on yarn-client.